packages/core/js-peer/src/js-peer/FluencePeer.ts

```diff
@@ -465,7 +462,16 @@ export class FluencePeer implements IFluenceClient {
         this._incomingParticles
             .pipe(
                 tap((x) => {
-                    x.particle.logTo('debug', 'particle received:');
+                    log.trace('id %s. received:', x.particle.id);
+                    log.debug('id %s. data: %j', x.particle.id, {
+                        initPeerId: x.particle.initPeerId,
+                        timestamp: x.particle.timestamp,
+                        tttl: x.particle.ttl,
+                        signature: x.particle.signature,
+                    });
+
+                    log.debug('id %s. script: %s', x.particle.id, x.particle.script);
```

Comment:
that's gonna clutter a lot. we have huuuuge scripts.
```suggestion
                    log.trace('id %s. script: %s', x.particle.id, x.particle.script);
```